frodo-android-sample/src/test/java/com/fernandocejas/example/frodo/sample/ObservableSampleTest.java

```diff
@@ -0,0 +1,23 @@
+package com.fernandocejas.example.frodo.sample;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import rx.Observable;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(RobolectricTestRunner.class)
+public class ObservableSampleTest {
+
+    @Test
+    public void returnsObservableInteger() throws Exception {
+        //given
```

Comment:
I am following Spock syntax with BDD labels.

You're the boss. I will remove comments.